docs/userdoc/Building.md

```diff
@@ -51,6 +51,18 @@ Rule based tuning can be enabled using a CMake option:
 cmake -DAUTOPAS_ENABLE_RULE_BASED_TUNING=ON .. 
 ```
 
+### Energy Measurements and Tuning
+
+By default, AutoPas tunes for the best configuration according to runtime. For some systems, it is also possible to tune
+for the algorithm that consumes the least energy. This is implemented via [Intel's RAPL](https://www.intel.com/content/www/us/en/developer/articles/technical/software-security-guidance/advisory-guidance/running-average-power-limit-energy-reporting.html) 
+(Running Average Power Limit) interface.
+
+Energy tuning can be enabled using a CMake option:
+```bash
+cmake -DAUTOPAS_ENABLE_ENERGY_MEASUREMENTS=ON .. 
+```
+*Note: Energy measurements are only available for x86 processors running a Linux kernel.*
```

Comment:
I don't really care about the note, but I would like to understand what you mean.  
Regarding the architecture, I cannot find an official comprehensive list of supported processors, but I also don't find a single source stating that the RAPL interface is available for a non-x86 machine. I'm rather confident that at the moment it is exclusively possible on x86, but of course this can change in the near future.

[Best list I could find](https://web.eece.maine.edu/~vweaver/projects/rapl/rapl_support.html)